lib/kubeclient/version.rb

```diff
@@ -2,5 +2,5 @@
 
 # Kubernetes REST-API Client
 module Kubeclient
-  VERSION = '4.9.2'
+  VERSION = '4.9.3'
```

Comment:
please drop the version bump from here.

I agree that token refresh feels worthy of a 4.10.z release :+1:, but current master has backward-incompatible changes not releasable as 4.y, and switch from rest-client to faraday makes this non-trivial to backport.
Need to see what we can do about it :thinking:, but meanwhile let's get it working on master...

@grosser could you review the rest?